bin/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Alternatively you can use override in the root config file.

Reply:
True! And we have precedent for that already as well.

Reply:
> Alternatively you can use override in the root config file.

Updated in 7f06252e4d.